images/kcp-upgrade/upgrade.sh

```diff
@@ -22,12 +22,20 @@ SCRIPT_DIR="$(
   cd "$(dirname "$0")" >/dev/null
   pwd
 )"
+
 CONFIG="$(dirname "$(dirname "$SCRIPT_DIR")")/config/config.yaml"
 current_kcp_version="$(yq '.version.kcp' "$CONFIG" | sed 's/v//' )"
 
 latest_kcp_version="$(curl -s https://api.github.com/repos/kcp-dev/kcp/releases/latest | yq '.tag_name' | sed 's/v//' )"
 
+if [[ -z "$latest_kcp_version" ]] || [[ -z "$current_kcp_version" ]]; then
+  printf "Something went wrong." >&2
```

Comment:
That's not a very helpful message. I'd prefer something along the lines of `"[ERROR] Could not retrieve kcp version: current='%s' latest='%s'" "${current_kcp_version:-not found}" "${latest_kcp_version:-not found}"` (please check the syntax if you copy/paste this).